### DISCUSSION TITLE
OpenAI Rate Limit exceeded? how can I fix this?

### DISCUSSION THREAD

**Discussion by MidoN37** [n3d1117/chatgpt-telegram-bot#187] [Q&A]
As the title says, whenever I try and chat with the bot it just spits out this: ⚠️ _OpenAI Rate Limit exceeded_ ⚠️
You exceeded your current quota, please check your plan and billing details.

I have no idea what this all means. what plan or details? I thought at least a few questions should be answered by the bot? or I'm I completely in the wrong here?

**Comment by n3d1117** [comment]
Hi @MidoN37 you can see your usage here: https://platform.openai.com/account/usage

**Reply by MidoN37** [reply]
I did, 18$ unused and the API is also never used. I double checked that the API I pasted on the env file is the one I have.

**Reply by n3d1117** [reply]
Make sure the 18$ credit is not expired. Are you able to use ChatGPT from the playground website? https://platform.openai.com/playground?mode=chat

**Reply by MidoN37** [reply]
Oh, Yep it's expired. I'll try from another account and let you know. Thank you!
